src/middlewares/prometheus_middleware.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+const excludedPaths = ['/health', '/metrics'];
+
+function prometheusMiddleware(promClient, register) {
+  const httpRequestDurationSeconds = new promClient.Histogram({
+    register: [register],
+    name: 'http_request_duration_seconds',
+    help: 'Request duration in seconds',
+    buckets: promClient.linearBuckets(0.1, 0.2, 20),
+    labelNames: ['path', 'status']
+  });
+
+  return function monitor(req, res, next) {
```

Comment:
and here: `return function monitor(req, res, next) {` => `return (req, res, next) => {`